storages/storages.go

```diff
@@ -94,11 +98,9 @@ func newCluster(name string, backendNames []string, backends map[string]http.Rou
 	}
 	for _, backendName := range backendNames {
 		backendRT, ok := backends[backendName]
-		if !ok {
-			return nil, fmt.Errorf("no such backend %q in 'storages::newCluster'", backendName)
+		if ok {
+			clusterBackends = append(clusterBackends, backendRT)
 		}
```

Comment:
Add "else block" which returs error as on master. If there is no required backend name registered in backends map it's obviously incorrect setup and we should fail